waspc/data/Generator/templates/react-app/src/auth/email/actions/passwordReset.ts

```diff
@@ -0,0 +1,20 @@
+{{={= =}=}}
+import api, { handleApiError } from '../../../api';
+
+export async function requestPasswordReset(data: { email: string; }): Promise<{ success: boolean }> {
+    try {
+        const response = await api.post('{= requestPasswordResetPath =}', data);
+        return response.data;
+    } catch (e: unknown) {
+        handleApiError(e);
+    }
+}
+
+export async function resetPassword(data: { token: string; password: string; }): Promise<{ success: boolean }> {
+    try {
+        const response = await api.post('{= resetPasswordPath =}', data);
+        return response;
```

Comment:
Just spot checking changed files. Does this need a `.data` too?

Reply:
Right you are 👍